### DISCUSSION TITLE
Border in line diagnostics popups

### DISCUSSION THREAD

**Discussion by Juliuzzk** [LazyVim/LazyVim#5280] [Q&A] [answered]
Hello, community!

Can someone help me? I’m not sure if this is caused by my configuration, but I’d like to know how I can fix or debug it.

![image](https://github.com/user-attachments/assets/36067cde-ddbd-4622-9709-5695b33fcf68)

i want to add a border like the next one

![image](https://github.com/user-attachments/assets/296db2cc-68ee-4693-8eef-9ba7a2a99d07)

Thanks you for your time.
Greetings!

**Comment by dpetka2001** [comment]
The upper arrow is an extmark that shows the diagnostic at the end of the line and I don't think it can have borders (but I might be wrong about it, since I've no knowledge about extmarks internals). In case I'm wrong, maybe someone else has a better solution for your case.

As far as the lower arrow is concerned, which is triggered when you do `<leader>cd`, you can do the following to add border
```lua
return {
	{
		"neovim/nvim-lspconfig",
		opts = {
			diagnostics = {
				float = {
					border = "rounded",
				},
			},
		},
	},
}
```

**Reply by Juliuzzk** [reply, marked as answer]
> The upper arrow is an extmark that shows the diagnostic at the end of the line and I don't think it can have borders (but I might be wrong about it, since I've no knowledge about extmarks internals). In case I'm wrong, maybe someone else has a better solution for your case.
> 
> As far as the lower arrow is concerned, which is triggered when you do `<leader>cd`, you can do the following to add border
> 
> ```lua
> return {
> 	{
> 		"neovim/nvim-lspconfig",
> 		opts = {
> 			diagnostics = {
> 				float = {
> 					border = "rounded",
> 				},
> 			},
> 		},
> 	},
> }
> ```

It worked, thank you so much!!

![image](https://github.com/user-attachments/assets/f0c6c92b-df03-4d54-bebf-e75c1ac14a54)
